modules/database/audit-autoloaded-options/update-autoload.js

```diff
@@ -0,0 +1,36 @@
+/* eslint camelcase: "off", no-alert: "off" */
+/* global jQuery:false, perflabAutoloadSettings:false */
+
+( function ( $, document ) {
+	$( document ).on( 'click', '.update-autoload', function ( e ) {
+		e.preventDefault();
+		const button = $( this );
+		const optionName = button.data( 'option-name' );
+		const autoload = button.data( 'autoload' );
+		const value = button.data( 'value' );
+		const data = {
+			option_name: optionName,
+			autoload,
+			value,
+		};
+
+		$.ajax( {
+			url:
+				perflabAutoloadSettings.root +
+				'perflab-aao/v1/update-autoload/' +
+				optionName,
+			method: 'POST',
+			data,
+			success( response ) {
+				if ( response.success ) {
+					button.attr( 'disabled', true );
+				} else {
+					alert( response.message );
+				}
+			},
+			error( error ) {
+				alert( error.errorMessage );
```

Comment:
Can we use something other than `alert()` for the success/error messages. It doesn't have a great UX.

Reply:
Instead of displaying an `alert`, I append that error message after the button. Could you please take a look?